pkg/asset/internal/templates.go

```diff
@@ -374,6 +374,7 @@ spec:
         image: {{ .Images.PodCheckpointer }}
         command:
         - /checkpoint
+        - --in-cluster=true
```

Comment:
Maybe I missed something, but how will the statically running version flip over to --in-cluster=false?